Find a functionally equivalent code:com.android.internal.policy.impl.PhoneWindow.initializePanelDecor:COMMENT
Method Modifier: protected   internal    
Comment:/**
 * Perform initial setup of a panel. This should at the very least set the
 * style information in the PanelFeatureState and must set
 * PanelFeatureState.decor to the panel's window decor view.
 *
 * @param st The panel being initialized.
 */

@@ -2,6 +2,12 @@
     st.decorView = new DecorView(getContext(), st.featureId);
     st.gravity = Gravity.CENTER | Gravity.BOTTOM;
     st.setStyle(getContext());
+    TypedArray a = getContext().obtainStyledAttributes(null, R.styleable.Window, 0, st.listPresenterTheme);
+    final float elevation = a.getDimension(R.styleable.Window_windowElevation, 0);
+    if (elevation != 0) {
+        st.decorView.setElevation(elevation);
+    }
+    a.recycle();
     return true;
 }

